forge-gui/res/cardsfolder/upcoming/braids_frightful_return.txt

```diff
@@ -2,10 +2,10 @@ Name:Braids's Frightful Return
 ManaCost:2 B
```

Comment:
I think the file name should be `braidss_frightful_return.txt` as ugly as that looks